fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/FedoraTypesUtils.java

```diff
@@ -129,18 +129,11 @@ public boolean apply(final Property p) {
             @Override
             public boolean apply(final Property p) {
                 try {
-                    if (!p.getDefinition().isProtected()) {
```

Comment:
I would suggest leaving the comment in the codebase, even if that means making this Predicate a private class.